moto/ec2/models.py

```diff
@@ -5947,11 +5961,12 @@ def create_from_cloudformation_json(
 
 
 class TransitGatewayBackend(object):
+
     def __init__(self):
         self.transit_gateways = {}
         super(TransitGatewayBackend, self).__init__()
 
-    def create_transit_gateway(self, description=None, options=None, tags=[]):
+    def create_transit_gateway(self, description=None, options={}, tags=[]):
```

Comment:
We need to be a bit careful here - let's better not use mutable values as default values for keyword arguments. The problem is that we may end up creating two distinct transit gateway objects with a reference to the exact same `options` object.

Better leave the parameter as `None`, and assign a default value in the method body itself. (In fact, it would be better to do the same for the `tags` as well).
```
options = options or {}
```

Reply:
Update the code.